git_machete/cmd.py

```diff
@@ -803,6 +806,74 @@ def slurp_file_or_empty(path: str) -> str:
     save_definition_file()
 
 
+def sync_before_pr(cli_ctxt: CommandLineContext) -> None:
```

Comment:
Hmmm actually... if you could prioritize rebasing your current changes onto current develop, for two reasons:
1. the more changes in more places you introduce on this PR, the harder it'll be to rebase later
2. once the state of `feature/hub` branch includes both `gh` CLI and your recent refactors, I can [dogfood](https://en.wikipedia.org/wiki/Eating_your_own_dog_food) both at the same time on my commercial project without shuttling b/w `develop` and `feature/hub` and reinstalling